tests/src/utils.rs

```diff
@@ -575,6 +586,7 @@ pub async fn run_exec(
     // minikube load image test:latest
     let mut base_env = HashMap::new();
     base_env.insert("MIRRORD_AGENT_IMAGE", "test");
+    base_env.insert("MIRRORD_AGENT_TTL", "30");
```

Comment:
Do we need the extra ttl? for logs or is there some other reason?

Reply:
Yeah was supposed to be for logs, but we removed the new test from CI anyways so I'll just remove that too.